src/internal.c

```diff
@@ -215,6 +217,12 @@ static int DoAppleNativeCertValidation(const WOLFSSL_BUFFER_INFO* certs,
                                             int totalCerts);
 #endif /* #if defined(__APPLE__) && defined(WOLFSSL_SYS_CA_CERTS) */
 
+#ifdef WOLFSSL_DTLS
+#ifndef DTLS_RECORDS_CAN_SPAN_DATAGRAMS_DEFAULT
+#define DTLS_RECORDS_CAN_SPAN_DATAGRAMS_DEFAULT 1
+#endif
+#endif
```

Comment:
What is this for? I don't see it used anywhere.

Reply:
good catch! removed.